src/redux/account/account.slice.ts

```
@@ -0,0 +1,37 @@
+    builder
+      // Get Profile Case
+      .addCase(getProfile.pending, (state: Draft<AccountState>) => {
+        state.loading.getProfile = true;
+        state.error = null;
+      })
+      .addCase(getProfile.fulfilled, (state: Draft<AccountState>, action: PayloadAction<GetProfileResponse>) => {
+        state.loading.getProfile = false;
+        state.error = null;
+        state.user = action.payload.data;
+      })
+      .addCase(getProfile.rejected, (state: Draft<AccountState>, action: PayloadAction<any>) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)